packages/theme/etc/theme.api.md

```diff
@@ -520,37 +520,37 @@ export interface ITheme extends Theme {
 // @public (undocumented)
 export namespace LocalizedFontFamilies {
     const // (undocumented)
-    Arabic: string;
+    Arabic = "'Segoe UI Web (Arabic)'";
```

Comment:
how come that this is now emitting exact type as string ? 

was this a bug within api-extractor ?

Reply:
something with resolving namespace, I think. Before it was producing two different results, now it produces consistent generated results for both, string and namespace

```ts
export const ChineseTraditional = `'Microsoft Jhenghei UI', Pmingliu`;
export const Cyrillic = `'${LocalizedFontNames.Cyrillic}'`;
```

Reply:
right, hmm this might be a breaking change for v8 👀